Numsense/Chinese.fs

```diff
@@ -0,0 +1,194 @@
+﻿module internal Ploeh.Numsense.Chinese
+
+open Ploeh.Numsense.InternalDsl
+
+type CharacterSet =
+    {
+        negative: string;
+        zero: string;
+        one: string;
+        two: string;
+        alternativeTwo: string;
+        three: string;
+        four: string;
+        five: string;
+        six: string;
+        seven: string;
+        eight: string;
+        nine: string;
+        ten: string;
+        hundred: string;
+        thousand: string;
+        tenThousands: string;
+        hundredMillions: string
+    }
+
+let internal TraditionalCharacterSet =
+    {
+        negative = "負";
+        zero = "零";
+        one = "一";
+        two = "二";
+        alternativeTwo = "兩";
+        three = "三";
+        four = "四";
+        five = "五";
+        six = "六";
+        seven = "七";
+        eight = "八";
+        nine = "九";
+        ten = "十";
+        hundred = "百";
+        thousand = "千";
+        tenThousands = "萬";
+        hundredMillions = "億"
+    }
+
+let internal TraditionalFinancialCharacterSet =
+    {
+        negative = "負";
+        zero = "零";
+        one = "壹";
+        two = "貳";
+        alternativeTwo = "兩";
+        three = "叄";
+        four = "肆";
+        five = "伍";
+        six = "陸";
+        seven = "柒";
+        eight = "捌";
+        nine = "玖";
+        ten = "拾";
+        hundred = "佰";
+        thousand = "仟";
+        tenThousands = "萬";
+        hundredMillions = "億"
+    }
+
+let internal SimplifiedCharacterSet =
+    {
+        negative = "负";
+        zero = "零";
+        one = "一";
+        two = "二";
+        alternativeTwo = "两";
+        three = "三";
+        four = "四";
+        five = "五";
+        six = "六";
+        seven = "七";
+        eight = "八";
+        nine = "九";
+        ten = "十";
+        hundred = "百";
+        thousand = "千";
+        tenThousands = "万";
+        hundredMillions = "亿"
+    }
+
+let internal SimplifiedFinancialCharacterSet =
+    {
+        negative = "负";
+        zero = "零";
+        one = "壹";
+        two = "贰";
+        alternativeTwo = "两";
+        three = "叁";
+        four = "肆";
+        five = "伍";
+        six = "陆";
+        seven = "柒";
+        eight = "捌";
+        nine = "玖";
+        ten = "拾";
+        hundred = "佰";
+        thousand = "仟";
+        tenThousands = "万";
+        hundredMillions = "亿"
+    }
+
+let rec internal toChineseImp useAlternativeTwo characterSet x =
```

Comment:
You are right, this is not needed here.

I added this parameter because in Chinese depending on few conditions a number two may be written in alternative form.
I think it is trivial for a user to replace the alternative form with the usual one so I assume it is better to have it always enabled.